### ISSUE TITLE
Training Scripts masked LM

### ISSUE THREAD

**Issue by friesel** [deepmind/deepmind-research#263]
Do you intend to publish the training scrips for the masked LM as well?

**Comment by diegolascasas** [comment]
Hi, can you specify which project you're directing your question to?

**Comment by friesel** [comment]
Sorry, my question is to the **Perceiver IO**-project-team. 

In the NLP-world often the pretrained models are just english or "all the worlds languages". Many users however need inference in non-english languages and have 1 or 2 GPUs rather than TPU-pods, so for them it's most efficient to pretrain only in the language you actually need inference in. So both for pretraining and finetuning it'd be great to have the **scripts you used in your pretraining of the masked LM** available. 

Thx

**Comment by comstockdonald** [comment]
I AM IN THE PROCESS OF FINDING OUT YOUR CREDENTIALS AND I WILL FOR SURE
THEN I WILL BRING YOU TO FEDERAL DOJ PROSECUTION WHO TOLD YOU TO TAMPER
WITH AND DESTROY FEDERAL AND INTRRNATIONAL DOCS AND FORMS YOU HAVE
DESTROYED TOO MUCH I SHOULD KNOW EVERYTHANGE ABOUT YOU IN THE NEXT 45 MIN
AND I WILL SEND AG NTS FOR YOU

**Comment by friesel** [comment]
Thx for the orientation. I will then get my head around the ImageNet-pipeline and try to adapt that to the NLP case.

**Comment by codedecde** [comment]
Hi @fding 
Would it be possible to share some of the tensorboard logs for the Byte level LM pretraining and/or specifics on what the final MLM loss the models converge to(something similar to https://github.com/google-research/electra/issues/3)? I am trying to replicate the Byte level experiments, so these logs would be really useful as a reference.
Thank you !
